src/java/picard/analysis/CollectMultipleMetrics.java

```diff
@@ -218,7 +282,20 @@ public static void main(final String[] args) {
         if (PROGRAM.isEmpty()) {
             return new String[]{"No programs specified with PROGRAM"};
         }
+        if ( (PROGRAM.contains(Program.CollectSequencingArtifactMetrics)) ) {
+            List<String> errs = new ArrayList<String>();
+            if (DB_SNP == null) {
+                errs.add("DB_SNP must be specified for CollectSequencingArtifactMetrics.");
+            }
+            if (INTERVALS == null) {
+                errs.add("INTERVALS must be specified for CollectSequencingArtifactMetrics.");
+            }
+            if (!errs.isEmpty()) {
+                return errs.toArray(new String[errs.size()]);
+            }
+        }
```

Comment:
One caveat: *sometimes* we'll want to pass in null values, so it would be better to not have the validation (yeah, pretty sketchy...). Sorry for not being clear about that.